src/SFA.DAS.ApplyService.Web/Services/RoatpOrganisationVerificationService.cs

```diff
@@ -91,6 +94,15 @@ private static bool WhosInControlStarted(JObject qnaApplicationData)
             return soleTraderOrPartnership == RoatpOrganisationTypes.SoleTrader || soleTraderOrPartnership == RoatpOrganisationTypes.Partnership;
         }
 
+        private async Task<bool> CharityCommissionDataExempted(JObject qnaApplicationData)
```

Comment:
Have you checked if the consumer has the ukprn handy and which can be passed in as a param here?

Reply:
It doesn't